public/src/components/channelManagement/bannerDesigns/BannerDesignForm.tsx

```diff
@@ -117,6 +118,17 @@ const BannerDesignForm: React.FC<Props> = ({
     });
   };
 
+  const onHeadlineSizeChange = (headerSize: 'small' | 'medium' | 'large' = 'medium'): void => {
```

Comment:
we can use the `FontSize` type here. 
Also, I don't think the default value is necessary since the `HeadlineSizeEditor` always provides a value

Reply:
Thanks - good spot